scripts/diff2html.py

```diff
@@ -60,6 +60,12 @@
 
 table_hdr = """
 <table class="table table-diff">
+<colgroup>
+  <col class="linenumber">
+  <col class="difftext">
+  <col class="linenumber">
+  <col class="difftext">
+</colgroup>
 """
```

Comment:
Since I included the change do `diff2html.py` here this should work when regenerating, right? Open for any pure-CSS version, this was just the only one I could get to work.